pytorch_lightning/trainer/training_loop.py

```diff
@@ -205,7 +205,7 @@ class TrainerTrainLoopMixin(ABC):
     check_val_every_n_epoch: ...
     num_training_batches: int
     val_check_batch: ...
-    num_val_batches: int
+    num_val_batches: List[int]
```

Comment:
this was renames as `limit_val_batches`

Reply:
sure? the code also has num_training_batches. I think the meaning here is different. 
Actually, here in this file num_val_batches is not even used. shall I remove it?

Reply:
well, my bad kind of... `num_training_batches` is still there but it is very confusing what si the diff to `limit_train_batches`

Reply:
num_training_batches is the final number of batches, whereas limit_train_batches can be both int and float (percentage).

Reply:
yeah, but from your description, you see that the functionality overlap...
cc: @williamFalcon shall we drop `num_training_batches`?

Reply:
ok, I removed the variable since it is not used in this file anyway.